app/client/ui/ProposedChangesPage.ts

```diff
@@ -0,0 +1,171 @@
+import { ActionLogPart, computeContext, showCell } from 'app/client/components/ActionLog';
+import { GristDoc } from 'app/client/components/GristDoc';
+import { testId } from 'app/client/lib/dom';
```

Comment:
```suggestion
import { testId } from 'app/client/ui2018/cssVars';
```

Just a suggestion to use the new style of testId (with classes not attributes), but it also requires test update.